README.md

```diff
@@ -1,2 +1,24 @@
 # hw01
 Repository for homework 1
+
+## Eigen Aoki
+
+Hello, my name is Eigen Aoki.
+
+![](http://www.taikolegacy.com/wp-content/uploads/2017/07/c_eigen_aoki-440x350.jpg)
+
+I'm a **MAPSS** student studying political theory. My thesis project looks at Guy Debord's *Society of the Spectacle* and compares it to today's political climate. It also asks what it means to be a "spectator" to politics through the works of Jacques Ranciere and Paulo Freire. 
+
+I'm originally from Oak Park, IL, just 30 minutes from Hyde Park. My undergraduate was at the University of Southern California where I majored in International Relations and East Asian Languages and Cultures. 
+
+###Hobbies and Misc.
+
+I like watching movies so here are a couple movies I saw recently that I enjoyed in no particular order. 
+
+1. Get Out
+2. The Handmaiden
+3. Perfect Blue
+
+As the photo above suggests, I am also a musician. I play bass and the Japanese Taiko Drums as a part of the Tsukasa Taiko group. Here is our website: http://www.taikolegacy.com/
```

Comment:
To improve presentation, you could also try adding a hyperlink to the word "website" with this syntax: [website] (http://www.taikolegacy.com) (minus the space between the [ and the parenthesis) 
It would look like this in preview: [website](http://www.taikolegacy.com/)